tests/test_similar.py

```diff
@@ -117,6 +119,24 @@ def test_duplicate_code_raw_strings_disable_scope(self) -> None:
             expected_output=expected_output,
         )
 
+    def test_duplicate_code_raw_strings_disable_scope_ignore_imports_and_signatures(
+        self,
+    ) -> None:
+        """Tests disabling duplicate-code at an inner scope level with
+        ignore_imports and ignore_signatures."""
```

Comment:
```suggestion
        ignore_imports and ignore_signatures.
        """
```

Weird, `pydocstringformatter` should catch this.